pom.xml

```diff
@@ -8,7 +8,7 @@
 
   <groupId>com.echobox</groupId>
   <artifactId>ebx-linkedin-sdk</artifactId>
-  <version>4.4.1</version>
+  <version>4.4.2</version>
```

Comment:
Version `4.4.1` hasn't been merged to master yet, so you can group all these changes together under version `4.4.1`. (Maybe just update the date in the changelog to today's date)

Sorry for the back and forth - I merged while you had this PR out for review :grimacing: